### ISSUE TITLE
scripts/delete-account.js in fxa-auth-server is broken again

### ISSUE THREAD

**Issue by lmorchard** [mozilla/fxa#5261]
I haven't been following the Typescript conversions on fxa-auth-server, but I think we might have broken `scripts/delete-account.js` again. ([Previously](https://github.com/mozilla/fxa/issues/4032)) A few other scripts affected too, like `clean-up-old-ci-stripe-customers.js`

Looks like `config/index.js` is now `config/index.ts`, which leads to this:
```
➜  fxa-auth-server git:(master) ✗ node scripts/delete-account.js lorchard@mozilla.com
internal/modules/cjs/loader.js:796
    throw err;
    ^

Error: Cannot find module '../config'
Require stack:
- /home/lmorchard/devel/fxa/packages/fxa-auth-server/scripts/delete-account.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:793:17)
    at Function.Module._load (internal/modules/cjs/loader.js:686:27)
    at Module.require (internal/modules/cjs/loader.js:848:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/home/lmorchard/devel/fxa/packages/fxa-auth-server/scripts/delete-account.js:25:16)
    at Module._compile (internal/modules/cjs/loader.js:955:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
    at Module.load (internal/modules/cjs/loader.js:811:32)
    at Function.Module._load (internal/modules/cjs/loader.js:723:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1043:10)
    at internal/main/run_main_module.js:17:11 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/lmorchard/devel/fxa/packages/fxa-auth-server/scripts/delete-account.js'
  ]
}
```

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-1857)
┆Issue Number: FXA-1857

**Comment by jodyheavener** [comment]
Here's a question for you @lmorchard (and @jackiemunroe as reviewer):

It looks like the most straightforward approach is to use `ts-node` (or maybe `npx ts-node`) to invoke the script(s), instead of `node`. No additional changes needed, we'd just need to communicate this to whoever uses this.

Is that reasonable? Or, should we attempt to either 1) wrap the script in a layer that, in itself, calls ts-node, or 2) ensure config.ts is compiled down to JS so the scripts can properly import it without needing to deal with TS. These two would permit whoever is executing the script to not change anything.

**Comment by lmorchard** [comment]
> It looks like the most straightforward approach is to use ts-node (or maybe npx ts-node) to invoke the script(s), instead of node. No additional changes needed, we'd just need to communicate this to whoever uses this.

Does that work for every script in `fxa-auth-server/scripts`? 

I think the most frequent users of `delete-account` will be folks like @jbuck and @jrgm - not sure who else will need to hear about a change here. I noticed `clean-up-old-ci-stripe-customers.js` is broken too when I tried to use it.

Not sure if there are others broken and/or if they all just work with a switch to ts-node.

Some thoughts off the top of my head that I haven't tried validating (😅):

Looks like these scripts have a shebang line of `#!/usr/bin/env node` - should probably change that to `ts-node`. I think that means it might make running the scripts as easy as e.g. `./delete-account.js`

Could also make some sense to rename the affected scripts to have `.ts` extensions to signify the change in script runner.

**Comment by jbuck** [comment]
Changing the shebang seems fairly straightforward to me to fix it

**Comment by lmorchard** [comment]
This issue isn't done being fixed, per comment on PR #5362 

> Also, per the initial issue, I ran delete-account.js and it additionally fails because it can't find ../lib/subhub/client. Sure enough, lib/subhub does not exist in the repo. Any ideas where this comes from?

**Comment by lmorchard** [comment]
Looks like the subhub bit was missed when I removed that stuff back in February:

https://github.com/mozilla/fxa/commit/d09125b8c8dd7a3363608c8a8b61446833b37804

Coincidentally, I think the last time this script got broken, it was when we *added* that.
